CHANGELOG.md

```diff
@@ -10,23 +10,55 @@
+
 
 ### Breaking changes
 
-- `StellarGraph.to_networkx` parameters now use `attr` to refer to NetworkX attributes, not `name` or `label` [\#973](http://github.com/stellargraph/stellargraph/pull/973). Migration: for any named parameters in `graph.to_networkx(...)`, change `node_type_name=...` to `node_type_attr=...` and similarly `edge_type_name` to `edge_type_attr`, `edge_weight_label` to `edge_weight_attr`, `feature_name` to `feature_attr`.
+- StellarGraph now requires TensorFlow 2.1 or greater, TensorFlow 2.0 is no longer supported [\#1008](https://github.com/stellargraph/stellargraph/pull/1008)
+- The legacy constructor using NetworkX graphs has been deprecated [\#1027](https://github.com/stellargraph/stellargraph/pull/1027). Migration: replace `StellarGraph(some_networkx_graph)` with `StellarGraph.from_networkx(some_networkx_graph)`, and similarly for `StellarDiGraph`.
+- Re-exports of calibration and ensembling functionality from the top-level of the `stellargraph` module were deprecated, in favour of importing from the `stellargraph.calibration` or `stellargraph.ensemble` submodules directly [\#1107](https://github.com/stellargraph/stellargraph/pull/1107). Migration: replace uses of `stellargraph.Ensemble` with `stellargraph.ensemble.Ensemble`, and similarly for the other names (see [\#1107](https://github.com/stellargraph/stellargraph/pull/1107) for all replacements).
+- `StellarGraph.to_networkx` parameters now use `attr` to refer to NetworkX attributes, not `name` or `label` [\#973](https://github.com/stellargraph/stellargraph/pull/973). Migration: for any named parameters in `graph.to_networkx(...)`, change `node_type_name=...` to `node_type_attr=...` and similarly `edge_type_name` to `edge_type_attr`, `edge_weight_label` to `edge_weight_attr`, `feature_name` to `feature_attr`.
+- `StellarGraph.nodes_of_type` is deprecated in favour of the `nodes` method [\#1111](https://github.com/stellargraph/stellargraph/pull/1111). Migration: replace `some_graph.nodes_of_type(some_type)` with `some_graph.nodes(node_type=some_type)`.
+- `StellarGraph.info`
+- Some more layers and models had many parameters move from `**kwargs` to real arguments: `Attri2Vec` ([\#1128](https://github.com/stellargraph/stellargraph/pull/1128)), `ClusterGCN` ([\#1129](https://github.com/stellargraph/stellargraph/pull/1129)), `GraphAttention` & `GAT` ([\#1130](https://github.com/stellargraph/stellargraph/pull/1130)), `HinSAGE` ([\#1143](https://github.com/stellargraph/stellargraph/pull/1143)). Invalid (e.g. incorrectly spelled) arguments would have been ignored previously, but now may fail with a `TypeError`; to fix, remove or correct the arguments.
+- The `method="chebyshev"` option to `FullBatchNodeGenerator`, `FullBatchLinkGenerator` and `GCN_Aadj_feats_op` has been removed, because it did not work [\#1028](https://github.com/stellargraph/stellargraph/pull/1028)
```

Comment:
Nice and to the point! ... but maybe this sounds slightly nicer
```suggestion
- The `method="chebyshev"` option to `FullBatchNodeGenerator`, `FullBatchLinkGenerator` and `GCN_Aadj_feats_op` has been removed for now, because it needed significant revision to be correctly implemented [\#1028](https://github.com/stellargraph/stellargraph/pull/1028)
```

Reply:
Yeah, that's much better 😄